File: src/server/gkill/api/req_res/get_idf_kyou_by_relative_path_response.go
package req_res

import "github.com/mt3hr/gkill/src/server/gkill/api/message"

type GetIDFKyouByRelativePathResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	// 見つからなかった場合は空文字
	KyouID string `json:"kyou_id"`
}

File: src/client/classes/api/req_res/get-idf-kyou-by-relative-path-response.ts
'use strict'

import { GkillAPIResponse } from '../gkill-api-response'

export class GetIDFKyouByRelativePathResponse extends GkillAPIResponse {

    // 見つからなかった場合は空文字
    kyou_id: string

    constructor() {
        super()
        this.kyou_id = ''
    }

}
